src/polyfillstub/intersection-observer-stub.js

```diff
@@ -26,6 +26,8 @@
 import {Services} from '../services';
 
 const UPGRADERS = '_upgraders';
+const NATIVE = 'AMP__NATIVE_INOB';
```

Comment:
Since we are not setting state on a global window, we can name these simpler. E.g. `_native` and `_stub`.